src/plugins/controls/public/react_controls/controls/data_controls/options_list_control/options_list_control_selections.ts

```diff
@@ -50,11 +50,11 @@ export function initializeOptionsListSelections(
     comparators: {
       exclude: [exclude$, setExclude],
       existsSelected: [existsSelected$, setExistsSelected],
-      selectedOptions: [selectedOptions$, setSelectedOptions, selectedOptionsComparatorFunction],
+      selections: [selectedOptions$, setSelectedOptions, selectedOptionsComparatorFunction],
     } as StateComparators<
-      Pick<OptionsListControlState, 'exclude' | 'existsSelected' | 'selectedOptions'>
+      Pick<OptionsListControlState, 'exclude' | 'existsSelected' | 'selections'>
     >,
-    hasInitialSelections: initialState.selectedOptions?.length || initialState.existsSelected,
+    hasInitialSelections: initialState.selections?.length || initialState.existsSelected,
     selectedOptions$: selectedOptions$ as PublishingSubject<OptionsListSelection[] | undefined>,
```

Comment:
Do we want to change `selectedOptions$` to be `selections$` to keep things consistent?

Reply:
I do like the name "selectedOptions" better as its more descriptive. To me "selections" is the entire set of state that users can make, such as "exists" and "exclude".

I do not have strong feelings about this though and am up for whatever the team decides. What do you think?

Reply:
Yeah, that's fair - I prefer `selectedOptions` too, but I wasn't sure if using `selections` for the "serialized state" but `selectedOptions` for the "runtime state" would cause confusion later. Maybe we leave a quick comment beside the type definition in `OptionsListControlState` for clarification? But it's probably not a huge deal - it's not a huge logical jump :)

Reply:
In runtime state, its also called "selections" since the comparator is keyed with "selections"